src/compiler/diagnosticMessages.json

```diff
@@ -2869,6 +2869,10 @@
         "category": "Error",
         "code": 6143
     },
+    "A package for '{0}' was found at '{1}', but is untyped. Because '--noImplicitAny' is enabled, this package must have a declaration.": {

```

Comment:
How about: `"Could not find a declaration file for module '{0}'. '{1}' implicitly has an 'any' type."`